pr-data.csv

```diff
@@ -1811,7 +1814,8 @@ https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,minifi/m
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,minifi/minifi-toolkit/minifi-toolkit-configuration,org.apache.nifi.minifi.toolkit.configuration.ConfigMainTest.testTransformVersionedFlowSnapshotSimple,ID,,,
 https://github.com/apache/nifi,12015a17dd93a1d42c9d6ddab6cc5ce606fef16a,nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-nar-utils,org.apache.nifi.nar.NarUnpackerTest.testUnpackNars,OD,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors,org.apache.nifi.processors.aws.wag.TestInvokeAmazonGatewayApiMock.testSendQueryParams,ID,,,
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-cybersecurity-bundle/nifi-cybersecurity-processors,org.apache.nifi.processors.cybersecurity.TestCompareFuzzyHash.testTLSHCompareFuzzyHashMultipleMatches,ID,,,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-cybersecurity-bundle/nifi-cybersecurity-processors,org.apache.nifi.processors.cybersecurity.TestCompareFuzzyHash.testTLSHCompareFuzzyHashMultipleMatches,ID,Accepted,https://github.com/apache/nifi/pull/6639,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-cybersecurity-bundle/nifi-cybersecurity-processors,org.apache.nifi.processors.cybersecurity.TestCompareFuzzyHash.testSsdeepCompareFuzzyHashMultipleMatches,ID,Accepted,https://github.com/apache/nifi/pull/6639,
```

Comment:
Same as above - put the SHA on which you found the test. You need not find when it was introduced, but it may have not even existed in `2bd752d868a8f3e36113b078bb576cf054e945e8`.

Reply:
I ran the commands for `2bd752d868a8f3e36113b078bb576cf054e945e8` and the flaky test existed in this SHA. Is it okay if I keep the SHA value as it is then?